ql/src/test/results/clientpositive/llap/subquery_select.q.out

```diff
@@ -3918,14 +3918,14 @@ STAGE PLANS:
                   Statistics: Num rows: 26 Data size: 208 Basic stats: COMPLETE Column stats: COMPLETE
                   Filter Operator
                     predicate: p_partkey BETWEEN 10000 AND 20000 (type: boolean)
-                    Statistics: Num rows: 8 Data size: 64 Basic stats: COMPLETE Column stats: COMPLETE
+                    Statistics: Num rows: 1 Data size: 8 Basic stats: COMPLETE Column stats: COMPLETE
                     Select Operator
                       expressions: p_size (type: int)
                       outputColumnNames: p_size
-                      Statistics: Num rows: 8 Data size: 64 Basic stats: COMPLETE Column stats: COMPLETE
+                      Statistics: Num rows: 1 Data size: 8 Basic stats: COMPLETE Column stats: COMPLETE
                       Group By Operator
                         aggregations: max(p_size)
-                        minReductionHashAggr: 0.875
+                        minReductionHashAggr: 0.0
```

Comment:
the computed range is most likely empty...
these changes suggest to me that something is not entirely right....is this expected?

Reply:
Range is `15103`-`195606` for `p_partkey` column, out of 26 rows. Hence, the estimate of `1` seems right.